examples/elm-pb/data/BOUT.inp

```diff
@@ -270,12 +263,7 @@ bndry_ydown = free_o3
 # Zero gradient in the core
 bndry_core = neumann
 
-[Vpar]
-
-bndry_core = neumann
-
 [phi]
 
 bndry_xin = none
 bndry_xout = none
-bndry_target = neumann
```

Comment:
This seems like quite a significant change, unless the y-boundary condition is actually set in the C++ code somewhere? Doesn't it now default to applying a Dirichlet bc?

Reply:
When running this case there was an error that this setting was not used. I think it's because the grid doesn't contain Y boundaries. I agree that if there are Y boundaries then this should definitely be included. Should we comment out that line instead, or mark this option as maybe used in elm-pb.cxx?

Reply:
If the option should always be `phi:bndry_target = neumann` when it's needed, the best might be to use the default-overriding feature (https://bout-dev.readthedocs.io/en/latest/user_docs/bout_options.html#overriding-library-defaults) and put
```
BOUT_OVERRIDE_DEFAULT_OPTION("phi:bndry_target", "neumann");
```
somewhere. I think either `elm-pb.cxx` or `elm-pb.hxx` should work - the statement needs to be outside of class or method definitions though. Then the other options can be left out of the input file.

Could be nice to do a similar thing with other options that should always be a certain thing, like `phi:bndry_xin`, etc. The less that needs to be in `BOUT.inp`, the less there is that could be wrong!